src/docs/style/typography.mdx

```diff
@@ -43,7 +43,7 @@ Unlike Material Design, the primary font for Brightlayer UI is [Open Sans](https
 
 # Type Scale
 
-Brightlayer UI provides a type scale that includes a range of styles to support the needs of your application. All styles and fonts are available in the design library. As Brightlayer UI continues to grow, additional font patterns will be added.
+Brightlayer UI provides a type scale that includes a range of styles to support the needs of your application. All styles and fonts are available in the design library. As Brightlayer UI continues to grow, additional font patterns will be added. See the [Angular](https://material.angular.io/guide/typography) and [React](https://mui.com/components/typography/) Material theming documentation for more details for how each framework handles typography.
```

Comment:
This can be easily Googled. Instead of this, I'd rather see a line at the bottom of the `TypographyTable` that goes,

> For developers using our themes, please refer to the documentation for Angular (link to BLUI ng theme readme), React (link to MUI typography page) and React Native (link to our React Native Typography component API) respectively.

Reply:
You may also get rid of "As Brightlayer UI continues to grow, additional font patterns will be added", as this is already covered by the contents in the "Customization" section below

Reply:
You got it, this is updated now.